extensions/amp-list/0.1/amp-list.js

```diff
@@ -602,6 +606,14 @@ export class AmpList extends AMP.BaseElement {
             'update': false,
           });
         }
+        toArray(
+          dev().assertElement(this.container_).children
+        ).forEach((child) =>
+          this.owners_./*OK*/ scheduleUnlayout(
+            dev().assertElement(this.container_),
+            child
+          )
+        );
```

Comment:
Under the hood `removeChildren` iterates through all the children already, so let's replace the call to `removeChildren` with similar logic:
```js
while (parent.firstChild) {
  this.owners_./*OK*/ scheduleUnlayout(
    dev().assertElement(parent),
    child
  )
  parent.removeChild(parent.firstChild);
}
```